README.md

```diff
@@ -66,16 +66,37 @@ about the supported options and format.
 For example, if we wanted to generate a blog with a `Post` model we could run the following command:
 
 ```bash
+# mix torch.gen.html <Context Module> <Schema Module> <Schema Table Name> [<Column Name>:<Column Type>]+
 $ mix torch.gen.html Blog Post posts title:string body:text published_at:datetime published:boolean views:integer
 ```
 
 The output would look like:
 
 ```bash
-Add the resource to your browser scope in lib/my_app_web/router.ex:
+* creating priv/templates/phx.gen.html/edit.html.heex
+* creating priv/templates/phx.gen.html/form.html.heex
+* creating priv/templates/phx.gen.html/index.html.heex
+* creating priv/templates/phx.gen.html/new.html.heex
+* creating priv/templates/phx.gen.html/show.html.heex
+* creating priv/templates/phx.gen.html/controller_test.exs
+* creating priv/templates/phx.gen.html/controller.ex
+* creating priv/templates/phx.gen.html/view.ex
+* creating priv/templates/phx.gen.context/access_no_schema.ex
+* creating priv/templates/phx.gen.context/context.ex
+* creating priv/templates/phx.gen.context/schema_access.ex
+* creating priv/templates/phx.gen.context/test_cases.exs
+* creating priv/templates/phx.gen.context/context_test.exs
+...<more output from phx.gen.html>...
```

Comment:
Can you cut out more of these lines, perhaps keeping only 2 or 3 of the top lines like I mentioned before, please?